assets/src/util/math.js

```diff
@@ -16,12 +16,26 @@ const median = arr => {
     : sorted[middle]
 }
 
+const isInRange = (data, lowerLimit, upperLimit) => {
+  if ((data >= lowerLimit) && (data <= upperLimit)) {
+    return true
+  } else { return false }
+}
+
+const isOutOfRange = (data, checkPointData) => {
+  if (data !== null && data <= checkPointData) {
```

Comment:
Okay, so just checking here, I know I commented on this: `checkPointData` is the first value considered to be out of range? Or is the last value considered to be in range?